modules/placement-driver/src/main/java/org/apache/ignite/internal/placementdriver/PlacementDriverManager.java

```diff
@@ -43,71 +79,114 @@
  * The another role of the manager is providing a node, which is leaseholder at the moment, for a particular replication group.
  */
 public class PlacementDriverManager implements IgniteComponent {
+    public static final String PLACEMENTDRIVER_PREFIX = "placementdriver.lease.";
+    /** Ignite logger. */
+    private static final IgniteLogger LOG = Loggers.forClass(PlacementDriverManager.class);
+    private static final long UPDATE_LEASE_MS = 200L;
+    public static final long LEASE_PERIOD = 10 * UPDATE_LEASE_MS;
     /** Busy lock to stop synchronously. */
     private final IgniteSpinBusyLock busyLock = new IgniteSpinBusyLock();
-
     private final RaftMessagesFactory raftMessagesFactory = new RaftMessagesFactory();
-
     /** Prevents double stopping of the component. */
     private final AtomicBoolean isStopped = new AtomicBoolean();
-
-    private final ReplicationGroupId replicationGroupId;
-
-    private final ClusterService clusterService;
-
-    private final Supplier<CompletableFuture<Set<String>>> placementDriverNodesNamesProvider;
-
+    /** Metastorage manager. */
+    private final MetaStorageManager metaStorageManager;
+    /**
+     * Vault manager.
+     */
+    private final VaultManager vaultManager;
     /**
-     * Raft client future. Can contain null, if this node is not in placement driver group.
+     * Configuration.
+     * TODO: The property is removed after assignments will moved to metastorage.
      */
+    private final TablesConfiguration tablesCfg;
+    private final HybridClock clock;
+    private final ClusterService clusterService;
+    private final ReplicationGroupId replicationGroupId;
+    private final Supplier<CompletableFuture<Set<String>>> placementDriverNodesNamesProvider;
+    /** Raft client future. Can contain null, if this node is not in placement driver group. */
```

Comment:
dont we leave empty lines between field definitions? seems that formally code style doesnt require this, but i got used to it too much )